augur/export_v2.py

```diff
@@ -767,6 +768,21 @@ def set_build_url(data_json, config, cmd_line_build_url):
     elif config.get("build_url"):
         data_json['meta']['build_url'] = config.get("build_url")
 
+def set_description(data_json, cmd_line_description_file):
+    """
+    Read Markdown file provided by *cmd_line_description_file* and set
+    `meta.description` in *data_json* to the text provided.
+    """
+    if not cmd_line_description_file.endswith('.md'):
+        fatal("Provided description file needs to be a Markdown file.")
```

Comment:
I think it's unfriendly to require a file extension when reading text input. The extension doesn't guarantee the contents of the file in any way. Requiring a specific extension only prevents alternative ways of providing a path to the option (e.g. `--description <(curl https://example.com/description.md)`).